cran-comments.md

```diff
@@ -1,6 +1,6 @@
 ## Test environments
 
-* local R installation, x86_64-apple-darwin17.0, R 4.1.3
+* local R installation, x86_64-apple-darwin17.0, R 4.2.0
```

Comment:
Should I add my local R installation as well, I am on 4.2.0 (Windows) as of last week ?